components/homme/src/test_mod.F90

```diff
@@ -77,7 +77,8 @@ subroutine set_test_initial_conditions(elem, deriv, hybrid, hvcoord, tl, nets, n
   endselect
 
   !initial conditions for initial run, runtype=0
-  if (runtype == 0) then
+  ! also does other test case setup.  
```

Comment:
Does this init code run for restart runs? What does the comment above, 'other test case', mean?

Reply:
i meant the other setup in addition to the initial condition.  some test cases (like supercell) initialize arrays that need to be done both on new runs and restart runs.